docs/Data_Portal/Users_Guide/Developers_Guide.md

```diff
@@ -1,75 +1,95 @@
 
-The GDC Portal contains an Analysis Center where applications are displayed for users to use with their cohorts.
-The GDC Portal also provides a framework for developing applications that can be used to analyze and visualize data from the GDC.
+The GDC Data Portal is built with the [React](https://reactjs.org/) framework and 
+the [Redux](https://redux.js.org/) library for state management. The GDC Data Portal uses [NextJS](https://nextjs.org/) as it's application framework which 
+provides server-side rendering of React components. [Mantine.dev](https://mantine.dev/) is the component library, and
+
+styling is through [TailwindCSS](https://tailwindcss.com/). The GDC Data Portal is built on top of the GDC API, which provides access to 
+the GDC data.
+
+![This image details the architecture of the GDC Data Portal. 
+It shows the interaction between the GDC Data API, the core
+module and the user interface.](./images/developers_guide/V2_architecture.png "Architecture of the GDC Data Portal")
+
+*Architecture of the GDC Data Portal*
 
 ## Overview of an Application
 
-Applications are High Order Components (HOC) that are rendered in the Analysis Center. The portal major functions
-like Project, Downloads, and Protein Paint are all applications. Each application handles a specific task and can be used to
-refine and analyze cohorts. Applications have access to all the current cohort information and can use that information
-to query the GDC API for additional information.
+Applications are React higher-order components (HOC) that are rendered in
+the [Analysis Center](https://portal.gdc.cancer.gov/analysis_page?app=). The GDC Data Portal major functions such as
+Projects, Repository,  and Protein Paint are all applications. Each application handles a specific task such as analysis or
+visualization and can also be used to refine and build cohorts. Applications are cohort centric and can 
+query the GDC API for additional information.
 
-Local and Cohort filters are available to applications. Local filters are filters that are specific to the application and are used to refine the data that is displayed in the application. Local filters are those available from the GDC API and are typically not the
-most common. For example in the Mutation Frequency application, the local filters are the gene and mutation type filters. In the figure
-below the local filters are highlighted in yellow. These filters are used to refine the input cohort allowing users to
-drill down to specific genes and mutation types of interest in the cohort.
+Local and Global filters are available to applications. Local filters are filters that are specific to the application
+and are used to refine the data that is displayed in the application. Local filters are those available from the GDC API
+and are typically not the most common. For example in the Mutation Frequency application, the local filters are the gene
```

Comment:
qn. i don't quite understand the "not the most common" part. do you mean, e.g. that some cohort filters are waaaay more common, e.g. age?

Reply:
maybe more frequently/less frequently used?

Reply:
ah, maybe that would work, though the local filters in Repository are probably highly used. 
or we can avoid overthinking it and just remove mention of whether they are common or frequently used or whatever.

Reply:
Removing all mention of common/frequency is fine with me.